server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -592,7 +592,7 @@
      *
      * @return the message
      */
-    @Message(id = Message.NONE, value = "The git branch to use to get the server configuration. Default is 'master'")
+    @Message(id = Message.NONE, value = "The git branch to use to get the server configuration. Default is 'main'")
```

Comment:
Please revert this change. The default is 'master'.  This describes the name of a branch in a user controlled git repository, not the name of a branch in any repo under the control of the WildFly project. Changing the default is a breaking change and not something we should mix with the rest of the things in this PR.

Reply:
You're right, thanks, I passed it over.